Apps/WebClient/src/ClientApp/app/components/timeline/medication.vue

```diff
@@ -159,6 +166,17 @@ $radius: 15px;
             </b-collapse>
           </b-col>
         </b-row>
+        <b-row>
```

Comment:
Not necessary right now, but. How hard would it be to move all logic and styles to a different component? That way it can be reused in the future for imms or other type of data.

Reply:
Can look into that in the next ticket.